src/content/terraform/product-landing.json

```
@@ -1,20 +1,20 @@
 {
 	"hero": {
-		"heading": "Terraform Placeholder Duplicate Of Vault Content",
+		"heading": "Automate Infrastructure on Any Cloud",
 		"image": "https://place-hold.it/600x320"
 	},
 	"overview": {
 		"heading": "What is Terraform?",
-		"body": "Secure, store and tightly control access to tokens, passwords, certificates, encryption keys for protecting secrets and other sensitive data using a UI, CLI, or HTTP API.",
+		"body": "Terraform is an infrastructure as code tool that lets you build, change, and version infrastructure safely and efficiently. This includes low-level components like compute instances, storage, and networking; and high-level components like DNS entries and SaaS features.",
 		"cta": {
-			"text": "Learn more about Terraform features",
-			"url": "/vault/docs/what-is-vault"
+			"text": "Learn more",
+			"url": "/terraform/intro"
 		},
 		"image": "https://www.datocms-assets.com/2885/1662053785-dev-dot-product-landing-what-is-terraform-diagram.png"
 	},
 	"get_started": {
-		"heading": "Getting Started",
-		"body": "Use Terraform to securely store, access, and manage secrets and other sensitive data.",
+		"heading": "Get Started",
+		"body": "Follow a code-complete, hands on tutorial to learn the Terraform basics with your favorite infrastructure provider.",
```

```suggestion
		"body": "Follow a code-complete, hands-on tutorial to learn the Terraform basics with your favorite infrastructure provider.",
```

I'm torn about "provider", too. Maybe "platform" instead?

---

Also, is it appropriate to include "Terraform Cloud" in the list below? The item says "learn terraform basics with your favorite infrastructure provider". TFC isn't really an infrastructure provider.